chart/values.schema.json

```
@@ -711,6 +711,58 @@
                 }
             ]
         },
+        "enableBuiltInSecretEnvVars": {
+            "description": "Uses built-in secret values set as environment variables passed to Airflow. You should supply corresponding environment variables as ``extraEnv`` variables if you disable them here.",
+            "type": "object",
+            "x-docsSection": "Airflow",
+            "properties": {
+                "AIRFLOW__CORE_FERNET_KEY": {
+                    "description": "Enable Airflow fernet key variable read from .Values.fernetKeySecretName",
+                    "type": "boolean",
+                    "default": true
+                },
+                "AIRFLOW__CORE__SQL_ALCHEMY_CONN": {
+                    "description": "Enable Airflow meta-data connection variable read from .Values.data.metadataSecretName",
+                    "type": "boolean",
+                    "default": true
+                },
+                "AIRFLOW_CONN_AIRFLOW_DB": {
+                    "description": "Enable Airflow meta-data connection variable read from .Values.data.metadataSecretName (Airflow 1.10 variant)",
+                    "type": "boolean",
+                    "default": true
+                },
+                "AIRFLOW__WEBSERVER__SECRET_KEY": {
+                    "description": "Enable Webserver secret key variable read from .Values.webserverSecretKeySecretName",
+                    "type": "boolean",
+                    "default": true
+                },
+                "AIRFLOW__CELERY__CELERY_RESULT_BACKEND": {
+                    "description": "Enable Celery backend url variable read from .Values.data.resultBackendSecretName",
+                    "type": "boolean",
+                    "default": true
+                },
+                "AIRFLOW__CELERY__RESULT_BACKEND": {
+                    "description": "Enable Celery backend url variable read from .Values.data.resultBackendSecretName (Airflow 1.10 variant)",
```

```suggestion
                    "description": "Enable Celery backend url variable read from .Values.data.resultBackendSecretName",
```

This isn't the earlier variant, the other one is.